pallets/liquidity-pools-gateway/routers/src/lib.rs

```diff
@@ -60,6 +60,9 @@ type CurrencyIdOf<T> = <T as pallet_xcm_transactor::Config>::CurrencyId;
 type MessageOf<T> = <T as pallet_liquidity_pools_gateway::Config>::Message;
 type AccountIdOf<T> = <T as frame_system::Config>::AccountId;
 
+/// Maximum size allowed for a byte representation of an EVM chain string.
+pub const MAX_EVM_CHAIN_SIZE: u32 = 16;
```

Comment:
Given that we should support the strings that Axelar is using, I think this might suffice, please let me know otherwise.

Reply:
Note that this is a string that _only_ makes sense for Axelar, AFAIK. I see value in being more specific regarding that, either in the naming of this const or in the place where it is defined.

Reply:
Will change, thank you!